tests/test_archive_readme.py

```diff
@@ -3,35 +3,74 @@
 We only run some smoke tests here.
 """
 
-import collections
 import os
 import tempfile
 
+import pytest
+
+from cubi_tk.__main__ import setup_argparse, main
+
 import cubi_tk.archive.readme
 
 
+def test_run_archive_readme_help(capsys):
+    parser, _subparsers = setup_argparse()
+    with pytest.raises(SystemExit) as e:
+        parser.parse_args(["archive", "readme", "--help"])
+
+    assert e.value.code == 0
+
+    res = capsys.readouterr()
+    assert res.out
+    assert not res.err
+
+
+def test_run_archive_readme_nothing(capsys):
+    parser, _subparsers = setup_argparse()
+
+    with pytest.raises(SystemExit) as e:
+        parser.parse_args(["archive", "readme"])
+
+    assert e.value.code == 2
+
+    res = capsys.readouterr()
+    assert not res.out
+    assert res.err
+
+
 def test_run_archive_readme_smoke_test():
+    cubi_tk.archive.readme.NO_INPUT = True
```

Comment:
Can you define this in the Constructor somehow? Or just calling like before, as a parameter?

Reply:
Please remove this line. Not necessary anymore.